### ISSUE TITLE
Dual Z Home behavior change intended?

### ISSUE THREAD

**Issue by cwhinton** [intelligent-agent/redeem#200]
The homing behavior on my dual Z CoreXY has changed with the recent RC (Pocahontas) vs. what I ran a year or so ago.  

Prior behavior was that both of the steppers on Z would be homed, thus ensuing the bed was level.  New behavior is home is stopped when the Z end stop is triggered, thus the bed could be out of level.

In my case H is slaved to Z and end stops Z and X2 are used.  This was tested with homing to Z-max as well as Z-min.

**Comment by Wackerbarth** [comment]
Yes, I have been complaining about this for over a year. The problem is that the new movement code handles only a limited aspect of "slaved" axes. You and I need them to behave as slaved during normal motion, but allow them to act independently for homing and as a part of a "leveling" adjustment.

In effect, what is wanted is for the Z-H pair to act as a CoreZH pair where "Z+H" = height*2 and "Z-H" can be zeroed. However, there is a constraint that "Z-H" remain "small", even when homing.
